src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/timeline/TimelineEntity.kt

```diff
@@ -20,34 +19,30 @@ import java.util.UUID
 
 @Table(name = "timeline")
 @Entity
-class TimelineEntity(
-  @Id
-  @GeneratedValue
-  @UuidGenerator
-  var id: UUID? = null,
-
+data class TimelineEntity(
   @Column(updatable = false)
-  @field:NotNull
-  var reference: UUID? = null,
+  val reference: UUID,
```

Comment:
I've also corrected the cardinality of each field - IE. use the kotlin language as intended and make then non-nullable (and `val` for fields that we do not expect/support changes to be made to)